.github/workflows/schools_updater.yml

```diff
@@ -0,0 +1,72 @@
+name: Fetch Paginated JSON and Update CSV
+
+on:
+  schedule:
+    - cron: "0 0 * * *"  # Runs once a day at midnight (UTC)
+  workflow_dispatch:    # Allows for manual runs as well
+  push:
+    branches:
+      - aa_schools_csv_updater
+
```

Comment:
Remove before merging.

```suggestion

```

Reply:
Leaving this as aa_schools_csv_updater for now as we want to ensure this is working before it hits main